pkg/storage/bucket/azure/config_test.go

```diff
@@ -13,7 +13,8 @@ import (
 
 // defaultConfig should match the default flag values defined in RegisterFlagsWithPrefix.
 var defaultConfig = Config{
-	MaxRetries: 20,
+	ContainerName: "cortex",
```

Comment:
nit: Why `cortex`? Wouldn't `loki` make more sense? ;-)

Reply:
This is the default that was already set in the code where the value was actually being used, I've just made sure it's consistently set so this test needed changing.

Reply:
I am interested as to why this is `cortex` but based on this and some of the comments I assume it was left over when the code was taken from Cortex?

Reply:
Yes, it's very likely a leftover.

Reply:
I guess the bigger question is why is this value added to the bucket config when it's only used in the chunk config?